package.json

```diff
@@ -82,6 +82,11 @@
       {
         "command": "gauge.execute.specification.all",
         "title": "Run All Specifications",
+        "category": "Gauge"
+      },
+      {
+        "command": "gauge.specexplorer.runAllSpecs",
```

Comment:
One Run all specifications command is hidden from the command palette. 

We do need to add a new command since we cannot send arguments to the command. When a command is invoked from a (context) menu, VS Code tries to infer the currently selected resource and passes that as a parameter when invoking the command but in this case we are not calling it from a context menu so there are no arguments which are passed by vscode. Therefore in the command scope we do not know whether the call came from the spec explorer or command palette and hence we added a new command and made it invisible in the command palette.